### DISCUSSION TITLE
How to enable Gamemode feature on Flatpak build

### DISCUSSION THREAD

**Discussion by chrsrns** [bottlesdevs/Bottles#637] [Q&A] [answered]
Hi. Gamemode is greyed out on my Flatpak install. Did I miss a step or is this really not available on Flatpak versions?

**Comment by mirkobrombin** [comment]
It should be available but doesnt' work. Flatpak has gamemode inside but the service is not working. What you can try is to install gamemode on your system, then start the service and expose it to the flatpak using flatseal.

**Reply by chrsrns** [reply]
Do I do this by exposing the directory where the `gamemode` and `gamemoderun` binaries are located?

**Reply by mirkobrombin** [reply]
Nope, Bottles need to see the service (systemctl status gamemode)

**Reply by chrsrns** [reply]
Ohh. How do I do that? Executing `systemctl status gamemode` in a terminal yields `Unit gamemode.service could not be found.`

**Reply by mirkobrombin** [reply]
Nevermind it require [access](https://github.com/bottlesdevs/Bottles/blob/dbcaa6a53256816931058e3e1de1730b2b70e3ef/src/backend/globals.py#L131) to the `gamemoderun` command but idk how to grant this permission, I'm also new in the Flatpak world ahahah

**Reply by chrsrns** [reply]
Bummer. Zap doesn't see the latest release right now for AppImage.

**Comment by q7nm** [comment, marked as answer]
Fixed.
